lib/ui/painting.dart

```diff
@@ -1741,6 +1741,16 @@ class ImageFilter extends NativeFieldWrapperClass2 {
     _initBlur(sigmaX, sigmaY);
   }
```

Comment:
let's remove the commented out ones above, now that we have more than one. They were just there to show the pattern.